webtool/test_transcribe.py

```
@@ -1627,6 +1646,31 @@ def _ki_projekt(monkeypatch, tmp_path, name, bases=("S1", "S2")):
+    still verschwinden. Genau eine durchgaengig stumme Attrappe nahm jedem Test den
+    Sensor fuer die [active]/[done]-Buchfuehrung; der erste #418-Fix lief daran
+    unbemerkt wirkungslos vorbei (14 Tests, alle gruen)."""
+    _ki_projekt(monkeypatch, tmp_path, "FixturDemo", diarize_druckt=True)
+    from webtool import correct
+    correct.cmd_diarize("FixturDemo", ["S1", "S2"])
+    out = capsys.readouterr().out
+    # PAAR je Datei (Bot an PR #493): nur [active] S1 + [done] S2 bliebe auch gruen, wenn
+    # S1 nie freigibt und S2 sich nie meldet — dann wuesste der Waechter nichts ueber
+    # die Paarform, an der die Buchfuehrung haengt.
+    for b in ("S1", "S2"):
+        assert f"[active] {b}" in out and f"[done] {b}" in out, out
```

Ruff (0.16.2)

[warning] Assertion should be broken down into multiple parts

(PT018)